lib/nx_zlib.c

```diff
@@ -1088,6 +1092,12 @@ void nx_hw_init(void)
 		nx_config.timeout_pgfaults = str_to_num(timeout_pgfaults);
 	}
 
+	if (soft_copy_threshold) {
+		nx_config.soft_copy_threshold = str_to_num(soft_copy_threshold);
+		nx_config.soft_copy_threshold = NX_MAX(nx_config.soft_copy_threshold, 31);
+		nx_config.soft_copy_threshold = NX_MIN(nx_config.soft_copy_threshold, 8190);
```

Comment:
Do we know which value is usually the best? If so we should default to it, otherwise this patches LGTM.

Reply:
@rzinsly, @tuliom has been doing some investigations on this threshold, but I don't think we have a "best" value so far. Right now if the user doesn't specify anything, it will keep using 1024 as the default like before.